app/src/features/FhirResourceTree/FhirResourceTree.tsx

```diff
@@ -95,6 +99,28 @@ const FhirResourceTree = (): JSX.Element => {
     }
   }, [expandedNodes, hasAlreadyExpandedToTarget, nodeAncestorsIds]);
 
+  const createStaticInputWithFixedValue = (
```

Comment:
I'd prefer if this function were `async` and that you `await` for the `createStaticInput` result